lib/http/base.js

```diff
@@ -51,8 +52,12 @@ HttpClient.prototype.request = function (options)
   var body;
   var request;
 
-  // normalize the headers
-  headers = normalizeHeaders(options.headers);
+  headers = {
+    ...normalizeHeaders(options.headers),
+    // axios does not know how to unzip response without content-length and  https://github.com/axios/axios/issues/5298
```

Comment:
Can you explain more? As far as I see, this issue you mention has been resolved for a few months.

Reply:
We found different root cause and applied different solution - check it out now